docs/usage.rst

```diff
@@ -231,6 +231,10 @@ Examples
         ~/src                             \
         --exclude '*.pyc'
 
+    # Backup home directories excluding directories containing image thumbnails
```

Comment:
maybe this should tell that only /home/*/.thumbnails is excluded, not /home/*/*/.thumbnails.